### DISCUSSION TITLE
Sonos Play1, Apple TV 4, Ds920+ and IOS Lock Screen?

### DISCUSSION THREAD

**Discussion by holderlibolder** [eizedev/AirConnect-Synology#41] [Q&A] [answered]
Hello,

i have airconnect running on my ds920+ DSM7.  I also use Sonos S1 with my 5 Play speakers. Although I shouted down all other apps on my phone I still don’t have Sonos on my Lock Screen. Am I missing something? Or isn’t this possible with airconnect? I appreciate any hint and ideas! Thanks!

**Comment by eizedev** [comment, marked as answer]
Hi, i also have no sonos on my lockscreen since iOS 12? 13? 
That problem is not related to AirConnect or AirPlay in general, i think this is a problem of the sonos app in combination with apples iOS.
